compose/web/bin/start-web

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -eou pipefail
+
+: ${APP_ENV?"APP_ENV not set. Aborting !"}
+: ${POSTGRES_HOST?"POSTGRES_HOST not set. Aborting !"}
+: ${POSTGRES_PORT?"POSTGRES_PORT not set. Aborting !"}
+: ${POSTGRES_USERNAME?"POSTGRES_USERNAME not set. Aborting !"}
+: ${POSTGRES_PASSWORD?"POSTGRES_PASSWORD not set. Aborting !"}
+: ${POSTGRES_DATABASE?"POSTGRES_DATABASE not set. Aborting !"}
+
+# prepare environment variable for django settings
+export DATABASE_URL="postgresql://$POSTGRES_USERNAME:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DATABASE"
+
+# setup pgpassfile for psql non-interactive authentication
+export PGPASSWORD="$POSTGRES_PASSWORD"
+
+# Setup Simpletable entries
+cat sql/simpletables/*.sql | psql -w -U "$POSTGRES_USERNAME"  -h "$POSTGRES_HOST"
+
+# Migrate
+python ./manage.py migrate --no-input
+
+# Import data for tables
+cat sql/*.sql | psql -w -U "$POSTGRES_USERNAME" -h "$POSTGRES_HOST"
```

Comment:
@cliftonmcintosh Line 18 - Line25 runs the database loading via sql files as well as runs django migrations. This is run within this `start-web` script before the django server starts. (So, basically whenever this container boots up and runs `start-web` the migrations/db restores are run.)
When a new sql file is added to the folders, one can simply `make stop` and `make start` which will load up all the new files.

Reply:
@suvash Thanks! I look forward to trying it. I should be able to run through it this weekend. Sorry for the delay, but weekdays are busy for me now.

Reply:
No worries. Let me know if something doesn't work etc. afterwards. 🍻